nodejs/device/src/index.ts

```diff
@@ -0,0 +1,318 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
>Licensed [](start = 3, length = 8)

Global comment: Promise objects are good for converting old (callback-style) code into new (async/await) code.  Do they serve any other purpose?  It seems like anything you could write with Promise object could be more clearly written with async/await.  It also seems like your code is hodge-podge of Promises and async functions.  My intuition says you should be consistent (unless you have a reason to not be consistent), but I'm not sure if I'm missing something.